engine/renderer/public/pipelines/fxaa_pipeline.hpp

```diff
@@ -0,0 +1,55 @@
+﻿#pragma once
+
+#include "frame_graph.hpp"
+#include "gbuffers.hpp"
+
+#include <cstdint>
+#include <memory>
+
+class BloomSettings;
+class GraphicsContext;
+class CameraResource;
+struct GPUImage;
+struct RenderSceneDescription;
```

Comment:
Forward declarations for BloomSettings and CameraResource are not being used

Reply:
Fixed